protocol-units/bridge/integration-tests/src/lib.rs

```diff
@@ -29,6 +42,41 @@ pub struct TestHarness {
 }
 
 impl TestHarness {
+
+	pub async fn new_with_movement() -> (Self, tokio::process::Child) {
+		let eth_client = EthClient::new(EthConfig::build_for_test())
+		.await
+		.expect("Failed to create EthClient");
+		let (movement_client, child) = MovementClient::new_for_test(MovementConfig::build_for_test())
+			.await
+			.expect("Failed to create MovementClient");
+		(
+			Self { eth_client: Some(eth_client), movement_client: Some(movement_client) 
```

Comment:
```suggestion
			Self { eth_client: None, movement_client: Some(movement_client) 
```
No need to build the eth_client on L:47 either, we just want this method to start up the test harness for the movement network